doc/api/vm.md

```diff
@@ -334,10 +390,10 @@ let code =
  vm.runInThisContext(code)(require);
  ```   
 
-_Note: `require()` in the above case shares the state with context it is passed
-from. This might introduce risks when unknown code is executed, e.g. altering
-objects from the calling thread's context in unwanted ways. It is advisable to
-run `vm` code in a separate process._
+*Note*: The `require()` in the above case shares the state with context it is
+passed from. This may introduce risks when untrusted code is executed, e.g.
+altering objects from the calling thread's context in unwanted ways. It is
+advisable to always run `vm` code within in a separate process.
```

Comment:
Yeah. I would prefer to just pull this out tbh
On May 18, 2016 5:37 AM, "Ben Noordhuis" <notifications@github.com> wrote:

> In doc/api/vm.md
> <https://github.com/nodejs/node/pull/6827#discussion_r63694453>:
>
> > @@ -334,10 +390,10 @@ let code =
> >   vm.runInThisContext(code)(require);
> >   ```
> >
> > -_Note: `require()` in the above case shares the state with context it is passed
> > -from. This might introduce risks when unknown code is executed, e.g. altering
> > -objects from the calling thread's context in unwanted ways. It is advisable to
> > -run `vm` code in a separate process._
> > +*Note*: The `require()` in the above case shares the state with context it is
> > +passed from. This may introduce risks when untrusted code is executed, e.g.
> > +altering objects from the calling thread's context in unwanted ways. It is
> > +advisable to always run `vm` code within in a separate process.
>
> As advice goes, this is only so-so. Running untrusted code in a separate
> process is no more secure when it runs with the same credentials as the
> current process.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6827/files/3eed279a0394e1c0927a90571c7bd41546a4ccff#r63694453>
>